scripts/gdb/lvglgdb/lvgl/misc/lv_image_cache.py

```diff
@@ -0,0 +1,88 @@
+from typing import Union
+import gdb
+from prettytable import PrettyTable
```

Comment:
Should we add a requirements.txt for dependencies.

Reply:
Great